docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +991,169 @@ eclagent:
+
+      <para>For more information about Kubernetes see the appropriate
+      Kubernetes documentation for your implementation.</para>
+
+      <sect3 id="Deploying_TheKubernetesSecret">
+        <title>Deploying the Kubernetes Secret</title>
+
+        <para>To deploy the Kubernetes secrets override the "secrets:" section
+        in HPCC-Platform/helm/hpcc/values.yaml, or deploy with your own
+        customized chart. For more information about customizing your HPCC
+        Systems containerized deployment see the above sections on
+        customization techniques.</para>
+
+        <para>In your chart, create a unique key name used to reference the
+        secret, and set it to the secret value that you created in the
+        previous step. In the above example it was
+        "admincredssecretname".</para>
+
+        <para>You can optionally define an additional alternate name, such as,
+        "admincredsaltmountname" which can be used with another Active
+        Directory server. The alternate name is illustrated in the example
+        below and should be added to your chart if necessary.</para>
+
+        <para>The "admincredsmountname" key/value pair already exists by
+        default in the HPCC Systems delivered values.yaml file. The key is
+        referenced in the component's ldap.yaml file. You may override these
+        and add additional key/values as needed.</para>
+
+        <programlisting lang="bash">  secrets:
+     authn:
+       admincredsmountname: "admincredssecretname"       #exernalize HPCC Admin creds
+       admincredsaltmountname: "admincredsaltsecretname" #exernalize alternate HPCC Admin creds
+</programlisting>
+      </sect3>
+
+      <sect3 id="ENABLE_k8S_LDAP">
+        <title>Enable LDAP Authentication</title>
+
+        <para>In the delivered
+        HPCC-Platform/esp/applications/common/ldap/ldap.yaml file, the
+        "ldapAdminSecretKey" is already set to the key mount name illustrated
+        in the example above. To enable LDAP authentication and to modify this
+        value, override the ESP/ECLWatch helm component located in values.yaml
+        as illustrated in the following example:</para>
+
+        <programlisting lang="YAML">esp:
+- name: eclwatch
+  application: eclwatch
+  auth: ldap
+  ldap:
+    ldapAddress: "myldapserver"
+    ldapAdminSecretKey: "admincredsaltmountname"   # use alternate secrets creds
+</programlisting>
+
+        <para> </para>
+      </sect3>
+    </sect2>
+
+    <sect2 id="Configuring_For_HashoCorpVault">
+      <title>Configuring LDAP in HashiCorp Vault</title>
+
+      <para>To create and store secrets in the HashiCorp Vault, from the
+      command command line interface, execute the following Vault commands.
+      The secret name used in the example below is "myvaultadmincreds" and
+      must be prefixed with "secret/authn/" as illustrated. The HPCC Systems
+      administrator "username" and "password" key/values are required; and
```

Comment:
"required; and additional ...". s/b "required. Additional properties are ignored"